is_core/rest/resource.py

```diff
@@ -256,3 +259,40 @@ def _generate_form_class(self, inst, exclude=[]):
         if hasattr(form_class, '_meta') and form_class._meta.exclude:
             exclude.extend(form_class._meta.exclude)
         return smartmodelform_factory(self.model, self.request, form=form_class, exclude=exclude, fields=fields)
+
+    def put(self):
+        return super(RestModelResource, self).put() if self.kwargs.get(self.pk_name) else self.update_bulk()
+
+    @transaction.atomic
+    def update_bulk(self):
+        qs = self._filter_queryset(self.core.get_queryset(self.request))
+        BULK_CHANGE_LIMIT = getattr(settings, 'BULK_CHANGE_LIMIT', 200)
+        if qs.count() > BULK_CHANGE_LIMIT:
+            return RestErrorResponse(
+                msg=ugettext('Only %s objects can be changed by one request').format(BULK_CHANGE_LIMIT),
+                code=413)
+
+        data = self.get_dict_data()
+        objects, errors = zip(*(self._update_obj(obj, data) for obj in qs))
+        compact_errors = tuple(err for err in errors if err)
+        return RestErrorsResponse(compact_errors) if len(compact_errors) > 0 else objects
+
+    def _update_obj(self, obj, data):
+        try:
+            return (self._create_or_update(merge(data, {self.pk_field_name: obj.pk})), None)
+        except DataInvalidException as ex:
+            return (None, self._format_message(obj, ex))
+        except (ConflictException, NotAllowedException):
```

Comment:
why do you catch those exception if you just reraise them a dont do anything with them?